### DISCUSSION TITLE
scripts jarvis interne Daten z.B. Icon, city, sunrise, sunset usw  nicht nutzbar

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2091] [Q&A]
Ich würde gern in scripts die internen Daten von jarvis nutzen. Sonst muss man alles selber nachmachen, was nicht sinnvoll ist.
Besteht die Möglichkeit die internen Daten als jarvis-Variablen freizugeben?
Wie kann man auf die Icons zugreifen in scripts?

**Comment by Zefau** [comment]
>  Besteht die Möglichkeit die internen Daten als jarvis-Variablen freizugeben?

Du meinst in Scripts? Das sollte gehen. Mach gerne einen Feature Request auf.

>  Wie kann man auf die Icons zugreifen in scripts?

Auf was willst du hier konkret zugreifen? Da müsstest du in die Iconify API schauen, was da möglich ist via HTTP-Request.
